site/metrics.md

```diff
@@ -20,6 +20,15 @@ exposed:
 * `weave_dns_entries` - Number of DNS entries.
 * `weave_flows` - Number of FastDP flows.
 
+#### Publish Router Metrics Endpoint
+
+By default weave only listens on it's local interface to serve metrics. 
+To publish your metrics throughout your cluster, e.g. if
+your prometheus server is installed on a different host machine,
+you need to set `WEAVE_STATUS_ADDR` to listen on all interfaces:
```

Comment:
If a user has multiple interfaces they may not not want to listen on interfaces exposed to the public Internet.  I don't think we want to flatly state they should set it to listen on all interfaces.